apps/test-mcp-server/eslint.config.mjs

```diff
@@ -1,4 +1,4 @@
-import config from "@tambo-ai-cloud/eslint-config/base";
+import config from "@tambo-ai/eslint-config/base";
 import tseslint from "typescript-eslint";
 
 export default tseslint.config(...config);
```

Comment:
The config helper is [deprecated](https://typescript-eslint.io/packages/typescript-eslint/#config-deprecated) in favor of eslint's `defineConfig`. Not critical for this PR but might be worthwhile swapping it out while you're in there?


```suggestion
import { defineConfig } from "eslint/config";
import config from "@tambo-ai/eslint-config/base";

export default defineConfig(...config);
```